### PR TITLE
CCPA Phase 1: Cookie Sync

### PR DESCRIPTION
**Pull request by SyntaxNode**
Implements: https://github.com/prebid/prebid-server/issues/1107

### CODE REVIEW

File: endpoints/auction.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/prebid/prebid-server/pbs"
 	"github.com/prebid/prebid-server/pbsmetrics"
 	pbc "github.com/prebid/prebid-server/prebid_cache_client"
+	"github.com/prebid/prebid-server/privacy"
+	gdprPolicy "github.com/prebid/prebid-server/privacy/gdpr"
```
Comment (SyntaxNode) [inline endpoints/auction.go]: Aliases such as this for gdpr will go away over time as the `gdpr` package is rolled into `privacy/gdpr`.

File: adapters/33across/usersync_test.go [hunk 2]
```diff
@@ -4,13 +4,19 @@ import (
 	"testing"
 	"text/template"
 
+	"github.com/prebid/prebid-server/privacy"
 	"github.com/stretchr/testify/assert"
 )
 
 func Test33AcrossSyncer(t *testing.T) {
-	temp := template.Must(template.New("sync-template").Parse("https://ic.tynt.com/r/d?m=xch&rt=html&ri=123&ru=%2Fsetuid%3Fbidder%3D33across%26uid%3D33XUSERID33X&id=zzz000000000002zzz"))
```
Comment (SyntaxNode) [inline adapters/33across/usersync_test.go]: This code is a bit hard to read. Unified to an easier to read pattern. I originally had changes to these files, but that's been removed but I thought the refactor is nice to have.

**Comment by hhhjort** [comment]
I think this is conflicting with the AMP PR.

**Comment by SyntaxNode** [comment]
> I think this is conflicting with the AMP PR.

Yeah. Not surprising. Fixing...

File: adapters/consumable/usersync_test.go [hunk 3]
```diff
@@ -4,17 +4,24 @@ import (
...
+	)
 
-	temp := template.Must(template.New("sync-template").Parse(
-		"//e.serverbid.com/udb/9969/match?redir=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dconsumable%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D"))
+	syncer := NewConsumableSyncer(syncURLTemplate)
+	u, _ := syncer.GetUsersyncInfo(privacy.Policies{
+		GDPR: gdpr.Policy{
+			Signal: "0",
+		},
+	})
```
[the remark this answers is not in the document]
Comment (SyntaxNode) [reply adapters/consumable/usersync_test.go]: Yes. I think it's a idea to keep it uniform. Fixed.

**Pull request merged**